docs/api/osquery-manager/saved-queries/create.asciidoc

```diff
@@ -59,7 +61,8 @@ $ curl -X POST api/osquery/saved_queries \
   "id": "saved_query_id",
   "description": "Saved query description",
   "query": "select * from uptime;",
-  "interval": "60",
+  "interval": 60,
```

Comment:
Hey @szwarckonrad, the description of `interval` says that it's an integer, but in the example it's formatted as a string. Just wanted to confirm which is correct.

Reply:
API expects `interval` field to be a string while in the UI it's being treated as a number.

Reply:
Got it, I'll update the description [here](https://github.com/elastic/kibana/pull/174595/files#diff-73bfd800ae21d3db5c5514cd3deae9cd7671daea2931a5358bd8a25589f10f80R38) to string then.

Reply:
Resolved in [b851c6b](https://github.com/elastic/kibana/pull/174595/commits/b851c6bd5c3aeb371bb5713a968b4c08519cce25).